test/expected.txt

```diff
@@ -135,6 +137,15 @@
 || 
 || 5 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  sort]
+|| 	1. Adam Brown Phone: 111111 Email: adam@gmail.com Address: 111, alpha street Tags: 
```

Comment:
It would be good to see an explicit test for sort.
E.g., add 2 people in non sorted order; list; sort; show that the address book is sorted.